test/end-to-end/test/test_importSanity.js

```diff
@@ -33,6 +43,7 @@ describe('Imported Content Sanity Testing', function () {
       const nightmare = new Nightmare();
       nightmare
         .goto(editRecipePage.url)
+        .wait(1000) // Have to wait for a while to make total count loaded
```

Comment:
Can we add  a more robust wait? For example in other frameworks (selenium or capybara+rspec) you can instruct the browser to wait until a specific condition is met, like an HTML element has been loaded for example. 

My problem with hard-coded timeouts is that they tend to expire and we have to constantly bump them up, which increases overall test execution time.